src/styles/_variables.scss

```diff
@@ -0,0 +1,4 @@
+$font-family: "Inter";
```

Comment:
Will you load Inter font from Google Fonts CDN?

Reply:
I thought about it when I wrote this exact line and then forgot about it, lol